lib/opal/nodes/top.rb

```diff
@@ -29,8 +29,8 @@ def compile
             add_temp '$nesting = []'
           end
           add_temp 'nil = Opal.nil'
-          add_temp '$$$ = Opal.const_get_qualified'
-          add_temp '$$ = Opal.const_get_relative'
+          add_temp '$$$ = Opal.$$$'
+          add_temp '$$ = Opal.$$'
 
```

Comment:
This will make the newly generated code incompatible with the 1.0.0 runtime. I would suggest bumping the version to like 1.0.1.

Reply:
@ahmadine you're right, actually master is targeted toward 1.1, while the 1.0 branch is `1-0-stable` and that will receive minor patches and bugfixes (actually I already released 1.0.2).

That said, it should still be compatible, as the local variable names won't change, except if someone was using stuff like `Opal.$$.Regexp.$$.IGNORECASE`.

_I'll update master to be v1.1.0.alpha shortly_

Reply:
I didn't mean "API" incompatibility, but "ABI" incompatibility. Otherwise, yeah, good to know, I missed the 1.0-stable branch.

Reply:
I see, yet, old compiled code should keep working (except for the `Opal.$$.Regexp…` example, which was only meant to aid debugging). Both `Opal.const_get_relative` and `Opal.const_get_qualified` are still available. Sorry if I ask again, it's because I want to be sure to really understand the implications of these changes 🙂

Reply:
New code uses `$$ = Opal.$$` which, when combined with the old runtime, would be equal to `$$ = undefined`. And that isn't going to work, because `Opal.$$ = Opal.const_get_qualified` is just defined in this changeset.

I think most people wouldn't be affected, but if someone combined the prebuilt CDN "binaries" with the master code which says "1.0.0", he could be surprised.

Reply:
Oh I see now, and yes I agree it shouldn't happen and it will be even less likely when I change the dev version to 1.1.0.alpha 👍